data/mundo/cpsAssets/23263889.json

```diff
@@ -395,7 +395,8 @@
            "tile":"Include from VisJo",
            "href":"/include/vjamericas/176-eclipse-lookup/mundo/app?responsive=true&newsapps=true&app-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-werty/venta-app_eclipse-mundo_bjd2w.png&app-clickable=true&amp-clickable=true&amp-image-height=360&amp-image-width=640&amp-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-werty/venta-app_eclipse-mundo_bjd2w.png",
            "platform":"highweb",
-           "type":"include"
+           "type":"include",
+           "url":"https://simorgh-include-test.s3-eu-west-1.amazonaws.com/vj.html"
```

Comment:
rather than putting this `url`s here with the pagedata which won't show up on test, should we default all VJ's to point to this and all IDT2's to point to the one below cc @jamesdonoh @andrewscfc @FK78 ??

Reply:
I'd favour that for now yeah